src/authentication/UserManagerMock.js

```diff
@@ -0,0 +1,30 @@
+export class UserManagerMock {
+    settings;
+
+    constructor(settings) {
+        this.settings = settings;
+    }
+
+    getUser() {
+        return Promise.resolve(JSON.parse(sessionStorage.getItem("user")));
+    }
+
+    signinRedirect() {
+        window.location = "/sign-in-callback";
+        return Promise.resolve(null);
+    }
+
+    signinSilent() {
+        return Promise.resolve(null);
+    }
+
+    signoutRedirect() {
+        sessionStorage.setItem("user", null);
```

Comment:
I think it's better to use a more specific test key. "powsybl-study-mock-user" ?